lang/en/validation.php

```diff
@@ -4,5 +4,5 @@
     'future_block'=> 'The :attribute must be at least :block.',
     'enough_token_supply' => 'The token supply is not enough.',
     'listing_not_cancelled' => 'The listing is already cancelled.',
-    'minimum_price' => 'The minimum bidding price is :price.',
+    'minimum_price' => 'The minimum bidding price must be greater than :price.',
```

Comment:
I think we can now say the bid needs to be greater than or equal to, as the price we show should work in a bid 👍🏻 
```suggestion
    'minimum_price' => 'The minimum bidding price must be greater than or equal to :price.',
```